cmd/minishift/cmd/image/config_view.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"github.com/spf13/cobra"
+
+	"fmt"
+	"github.com/minishift/minishift/cmd/minishift/cmd/config"
+	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"sort"
+	"strings"
+)
+
+var (
+	listConfiguredImagesCmd = &cobra.Command{
+		Use:   "view",
+		Short: "Displays the configured list of images for import and export.",
+		Long:  "Displays the configured list of images for import and export.",
+		Run:   viewConfiguredImages,
+	}
+)
+
+func viewConfiguredImages(cmd *cobra.Command, args []string) {
+	configMap, err := config.ReadConfig()
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Cannot read the Minishift configuration: %s", err.Error()))
+	}
+
+	if configMap[config.CacheImages.Name] == nil || len(configMap[config.CacheImages.Name].([]interface{})) == 0 {
```

Comment:
Candidate to be extracted into a utility function. This makes the actual command shorter, and moves all logic to deal with add, remove, list in the same place:
```
+	if configMap[config.CacheImages.Name] == nil || len(configMap[config.CacheImages.Name].([]interface{})) == 0 {
+		return
+	}
+
+	imageList := toStringSlice(configMap[config.CacheImages.Name].([]interface{}))
+	sort.Strings(imageList)
+	fmt.Println(strings.Join(imageList, "\n"))
```